openssl/ssl.zig

```diff
@@ -97,7 +97,7 @@ pub const SslConn = struct {
         {
             const result = openssl.SSL_connect(ssl);
             if (result != 1) {
-                std.debug.warn("SSL_connect failed with {}\n", .{result});
+                std.debug.print("SSL_connect failed with {d}\n", .{result});
```

Comment:
What's the benefit for specifying `d` here?

Reply:
No specific benefit, it just makes sure that it gets printed in decimal form.